lib/reports/middleware/authentication.rb

```diff
@@ -0,0 +1,19 @@
+module Reports
+  module Middleware
+    class Authentication < Faraday::Middleware
+      def initialize(app)
+        super(app)
+        @token = ENV['GITHUB_TOKEN']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.